concent_api/core/views.py

```diff
@@ -374,6 +403,17 @@ def validate_golem_message_reject(data):
             raise Http400("Wrong type of deadline field.")
 
 
+def validate_golem_message_timestamp(timestamp):
+    try:
+        verify_time(timestamp)
+    except MessageFromFutureError:
+        raise Http400('Message timestamp too much in the future.')
+    except MessageTooOldError:
+        raise Http400('Message timestamp too old.')
+    except TimestampError as e:
+        raise Http400(e)
```

Comment:
Please don't use single-letter names for your variables.

Reply:
:+1: